tox.ini

```
@@ -59,17 +59,56 @@ commands =
 deps =
     -r{toxinidir}/requirements/quality.txt
 commands =
-    black --check --diff --line-length=80 src/scriv tests docs setup.py
+    black --check --diff --line-length=80 src/scriv tests docs
     python -m cogapp -cP --check --verbosity=1 docs/*.rst
     mypy src/scriv tests
-    pylint src/scriv tests docs setup.py
-    pycodestyle src/scriv tests docs setup.py
-    pydocstyle src/scriv tests docs setup.py
-    isort --check-only --diff -p scriv tests src/scriv setup.py
-    python setup.py -q sdist
+    pylint src/scriv tests docs
+    pycodestyle src/scriv tests docs
+    pydocstyle src/scriv tests docs
+    isort --check-only --diff -p scriv tests src/scriv
+    python -m build --sdist
     twine check dist/*
```

It's not very useful unless strict:
```suggestion
    twine check --strict dist/*
```